File: ConnectionService/src/main/java/com/example/connectionsservice/Dto/FollowRequestsDTO.java
package com.example.connectionsservice.Dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class FollowRequestsDTO {
    public String followerId;
    public String toFollowId;
}

File: client-app/src/app/model/FollowRequestsDTO.ts
export interface FollowRequestsDTOInterface {
  followerId:String;
  toFollowId: String;
}
export class FollowRequestsDTO implements FollowRequestsDTOInterface {
  followerId:String;
  toFollowId: String;

  constructor(obj: FollowRequestsDTOInterface) {
    this.followerId = obj.followerId;
    this.toFollowId = obj.toFollowId;
  }
}
